ansible/roles/ocp4-workload-nfs-server/tasks/workload.yml

```diff
@@ -0,0 +1,202 @@
+---
+- name: Create or update storage class
+  k8s:
+    state: present
+    definition:
+      apiVersion: storage.k8s.io/v1
+      kind: StorageClass
+      metadata:
+        name: "{{ _nfs_provided_storage_class }}"
+      provisioner: kubernetes.io/no-provisioner
+      reclaimPolicy: Retain
+      volumeBindingMode: WaitForFirstConsumer
+
+- name: Create project for NFS server
+  k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ _nfs_project }}"
+
+- name: Create PVC for NFS server underlying storage
+  k8s:
+    state: present
+    definition:
+      kind: PersistentVolumeClaim
+      apiVersion: v1
+      metadata:
+        labels:
+          app: "{{ _nfs_app }}"
+        name: "{{ _nfs_app }}-pvc"
+        namespace: "{{ _nfs_project }}"
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        storageClassName: "{{ _nfs_pvc_type }}"
+        resources:
+          requests:
+            storage: "{{ _nfs_pvc_size }}"
+
+- name: Create ImageStream for nfs server image
+  k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: ImageStream
+      metadata:
+        labels:
+          app: "{{ _nfs_app }}"
+        name: "{{ _nfs_app }}"
+        namespace: "{{ _nfs_project }}"
+      spec: {}
+
+- name: Import image to ImageStream
+  shell: oc import-image --from={{ _nfs_server_image }} --scheduled {{ _nfs_app }} -n {{ _nfs_project }}
+
+- name: Create service account
+  k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        name: "{{ _nfs_app }}"
+        namespace: "{{ _nfs_project }}"
+
+- name: Add anyuid SCC to SA
+  shell: oc adm policy add-scc-to-user anyuid system:serviceaccount:{{ _nfs_project }}:{{ _nfs_app }}
```

Comment:
`s/shell/command`

Reply:
Why do you want shell instead of command? I have never heard a firm reason one way or the other.

Reply:
Because sometimes you just don't need a shell to run something.

you usually use `shell`  when you need features from the shell like:

- pipes
- variables
- `&`, `if/else`
- stream operation `>` `<`
- some variables like $HOME
- ...

If you just run a simple command like here, i'd prefer command.
I assume it's faster, and probably safer: shell is another potentially "tainted" environment that could go wrong.

Reply:
You could also just use k8s to create the role binding